README.md

```diff
@@ -238,6 +238,7 @@ The schema for the payload returned is:
 - `?tag=<string>` - Filter entries by a specific tag
 - `?issue=<string>` - Filter entries by an issue area
 - `?help_type=<string>` - Filter entries by a specific help category
+- `?has_help_types=<True or False>` - Filter entries by whether they have help types or not. Note that `True` or `False` is case-sensitive.
```

Comment:
Can you also document this in `entries/views.py` so it shows up at the top of `/api/pulse/entries`?

Reply:
done